pycbc/waveform/waveform.py

```diff
@@ -927,6 +927,36 @@ def get_waveform_filter_length_in_time(approximant, template=None, **kwargs):
     else:
         return None
 
+def get_waveform_from_ascii(waveform,epoch=0.0,Norm=5.9029581035870565e+20):
+    """Return the plus and cross polarizations of a time domain waveform.
+    Parameters
+    ----------
+    waveform: Path of the ASCII file containing time, hp, hc
+    reference_distance in Mpc of the generate waveform
+
+    Returns
+    -------
+    hplus: TimeSeries
+        The plus polarization of the waveform at distance and incliation
+    hcross: TimeSeries
+        The cross polarization of the waveform at distance and inclination
+    sample rate : retunrs the sample rate of the wavefom in the txt file 
+    """
+    infile=waveform
+    #'/home/shubhanshu.tiwari/cWB_Sergey_Shubhanshu_Vaibhav/cWB_share_m1-1.0000000e+01_m2-1.0000000e+01_ecc-3.0000000e-01.dat'
+    N=Norm # 5.9029581035870565e+20 ## at 100 Mpc change for other waveforms
+    data=numpy.loadtxt(infile)
+    t=data[:,0]
+    hp1=data[:,1]
+    hc1=data[:,2]
+    deltaT=(t[3]-t[2])
+    hp_ref=hp1/N
+    hc_ref=hc1/N
+    hp=TimeSeries(hp_ref, delta_t=deltaT, epoch=epoch)
+    hc=TimeSeries(hc_ref, delta_t=deltaT, epoch=epoch)
+    return hp, hc, 1./deltaT
```

Comment:
I don't think you need to return 1.0 / deltaT here. That information is already contained in the waveform themselves. I.E. as hp.duration